.cicd/test-package.anka.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -euo pipefail
+
+. "${0%/*}/libfunctions.sh"
+
+echo '--- :anka: Pretest Setup'
+
+if [[ ! $(brew --version 2>/dev/null) ]]; then
+   perform 'echo | /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"'
+   perform 'brew analytics off'
+fi
```

Comment:
I guess I missed this in the previous reviews. Homebrew is already installed on the Anka base layers and, even if it weren't, we should _never_ be piping `curl` into `bash`. There are a number of reasons why this is bad practice, but [here](https://www.idontplaydarts.com/2016/04/detecting-curl-pipe-bash-server-side/) is a particularly interesting one.

If we _have_ to blindly trust BASH scripts from the Internet, we should at least save and retain a local copy so that we can do forensic analysis on it if we are attacked.

We should also remove this code if it exists on the other branches.